Add WeightedPool ABIs to deployments package (#607)
balancer-labs/balancer-v2-monorepo: pkg/deployments/README.md

This package contains the addresses and ABIs of all Balancer V2 deployed contracts, for mainnet and various test networks. Artifacts are found in the [`deployed`](./deployed) directory, where each subdirectory represents a network.

> Note that some protocol contracts are created dynamically: for example, `WeightedPool` contracts are deployed by the canonical `WeightedPoolFactory`. The ABIs of these contracts are stored in the [`extra-abis`](./extra-abis) directory, but their addresses are not. Those should be instead retrieved by querying the on-chain state, or processing emitted events.

Reviewer: ```suggestion
Note that some protocol contracts are created dynamically: for example, `WeightedPool` contracts are deployed by the canonical `WeightedPoolFactory`. The ABIs of these contracts are stored in the [`extra-abis`](./extra-abis) directory, but their addresses are not. Those can be retrieved by querying the on-chain state, or processing emitted events.
```

import { ethers } from 'hardhat';
import { Contract } from 'ethers';

> // Creates an ethers Contract object for a canonical contract deployed at a network

Reviewer: ```suggestion
// Creates an ethers Contract object for a canonical contract deployed on a specific network
```
